workflow-templates/README.md

```diff
@@ -0,0 +1,11 @@
+# Workflow Templates
+
+This folder contains Workflow Templates made available to all TV 2 developers. To use these visit "Actions > New workflow" in your repo, where these templates will show up at the top of the list as "Owned by TV 2 Danmark".
```

Comment:
I wasn't sure what the exact wording of "Owned by ..." was, but I assume it's the name of the GitHub org? In that case it's TV 2 Danmark.

Reply:
Yes, it should be the display name of the organisation. I did test this on my own org prior to describing it, so you can verify it from this image:

<img width="645" height="664" alt="image" src="https://github.com/user-attachments/assets/43375fb9-de35-4087-8faa-5c7ee52c718e" />

My org is named `devantler-tech` and its display name is "DevantlerTech". For TV 2 we can verify this will be the name here:

<img width="824" height="230" alt="image" src="https://github.com/user-attachments/assets/ccdd93e3-6816-4cdf-9d1c-9a77c00d93e6" />

Reply:
Although it needs to say "By TV 2 Danmark", so I will that :-)